api/v1alpha1/shared_types.go

```diff
@@ -191,6 +191,13 @@ type KubernetesServiceSpec struct {
 	// +optional
 	AllocateLoadBalancerNodePorts *bool `json:"allocateLoadBalancerNodePorts,omitempty"`
 
+	// LoadBalancerIP defines the IP Address of the underlying load balancer service. This field
+	// may be ignored if the load balancer provider does not support this feature.
+	// This field has been Deprecated, but it is still used for set the IP Address in some cloud
```

Comment:
nit: `set` --> `setting`

Might also be good to clarify that it is deprecated in k8s so maybe `This field has been Deprecated in K8s`. 

If you wanted to get fancy you could add a link but since this gets bundled into the CRD definition, this seems sufficient. Also, I think we have good history in the PR that users can search if have questions.

Reply:
thanks, addressed